src/Umbraco.Core/Services/ContentEditingService.cs

```diff
@@ -36,16 +36,33 @@ public ContentEditingService(
         IUserService userService,
         ILocalizationService localizationService,
         ILanguageService languageService,
-        IOptions<ContentSettings> contentSettings)
-        : base(contentService, contentTypeService, propertyEditorCollection, dataTypeService, logger, scopeProvider, userIdKeyResolver, contentValidationService, treeEntitySortingService)
+        IOptionsMonitor<ContentSettings> optionsMonitor,
+        IRelationService relationService)
+        : base(
+            contentService,
+            contentTypeService,
+            propertyEditorCollection,
+            dataTypeService,
+            logger,
+            scopeProvider,
+            userIdKeyResolver,
+            contentValidationService,
+            treeEntitySortingService,
+            optionsMonitor,
+            relationService)
     {
         _propertyEditorCollection = propertyEditorCollection;
         _templateService = templateService;
         _logger = logger;
         _userService = userService;
         _localizationService = localizationService;
         _languageService = languageService;
-        _contentSettings = contentSettings.Value;
+        _contentSettings = optionsMonitor.CurrentValue;
+        optionsMonitor.OnChange((contentSettings) =>
+        {
+            _contentSettings = contentSettings;
+        });
+
```

Comment:
Could you expose `ContentSettings` as a `protected` property on `ContentEditingServiceBase`, instead of implementing the monitor in both places?

Reply:
Good point!